src/index.html

```diff
@@ -11,12 +11,115 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="stylesheet"
+      href="../../reset/reset.css"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <h1 class="visualy_hidden">Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <!--navigator-->
+        <ul class="nav__list">
+          <!--container for items of list-->
+          <li class="list__elem">
```

Comment:
Fix classes everywhere
```suggestion
          <li class="nav__item">
```